scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.ts

```diff
@@ -166,7 +175,7 @@ export class UseCasesComponent {
         }
     }
 
-    collapsePanel(event) {
+    collapsePanel(event: boolean) {
```

Comment:
event suggests a the type Event but it's something else. Probably same as the variable. isPanelCollapsed.